lib/helpers.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
+"""## Import needed packages
```

Comment:
The file docstring is little confusing. Because it's just various helpers it can be removed.